tests/ideas/test_idea_export_view.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+from django.core.exceptions import PermissionDenied
+
+from apps.ideas import views
+
+
+@pytest.mark.django_db
+def test_idea_sketch_export_view_user(rf, user):
+    view = views.IdeaExportView.as_view()
+    request = rf.get('/ideas/list/export')
+    request.user = user
+    with pytest.raises(PermissionDenied):
+        view(request)
+
+
+@pytest.mark.django_db
+def test_idea_export_view_admin(rf, admin, idea_sketch_factory,
+                                proposal_factory):
+    # the default filters without an active phase set 'status': 'winner'
+    idea_sketch = idea_sketch_factory()
+    idea_sketch.is_winner = True
```

Comment:
Use `idea_sketch = idea_sketch_factory(is_winner=True)` and save 3 lines, but only for a short while.